libs/utilities/src/tests/stringUtils.test.ts

```diff
@@ -0,0 +1,71 @@
+// (C) 2007-2020 GoodData Corporation
+import { randomString, shortenText, simplifyText, parseStringToArray, hashCodeString } from "../stringUtils";
+
+describe("#randomString", () => {
+    it("should generate alphanumeric string with correct length", () => {
+        expect(randomString(10)).toEqual(expect.stringMatching(/^[0-9a-zA-Z]{10}$/));
+    });
+});
+
+describe("#shortenText", () => {
+    it("should shorten text to maximum length and add ellipsis", () => {
+        const shortened = shortenText("Lorem ipsum dolor sit amet", { maxLength: 11 });
+        expect(shortened).toEqual("Lorem ipsum…");
+    });
+
+    it("should NOT shorten text when maxLength is not passed", () => {
+        const shortened = shortenText("Lorem ipsum dolor sit amet");
+        expect(shortened).toEqual("Lorem ipsum dolor sit amet");
+    });
+
+    it("should NOT shorten text when text is shorter than maxLength", () => {
+        const shortened = shortenText("Lorem ipsum dolor sit amet", { maxLength: 100 });
+        expect(shortened).toEqual("Lorem ipsum dolor sit amet");
+    });
+});
+
+describe("#simplifyText", () => {
+    it("should simplify text", () => {
+        expect(simplifyText("")).toEqual("");
+        expect(simplifyText(123)).toEqual("123");
+        expect(simplifyText(" - ")).toEqual("___");
+        expect(simplifyText("123456")).toEqual("123456");
+        expect(simplifyText("Too long")).toEqual("too_long");
+        expect(simplifyText(null)).toEqual("");
+    });
+});
+
+describe("parseStringToArray", () => {
+    it("should parse string to array correctly", () => {
```

Comment:
This should probably be a parametric test.

Reply:
fixed